wfl/fit/mace.py

```diff
@@ -0,0 +1,205 @@
+import os, sys, yaml, subprocess
+import warnings
+import ase.io
+from expyre import ExPyRe
+from wfl.configset import ConfigSet, OutputSpec
+from wfl.autoparallelize.utils import get_remote_info
+from expyre.resources import Resources
+from pathlib import Path 
+from shutil import copyfile
+
+
+def prepare_configs(fitting_configs):
+    """Prepare configs before fitting. Currently only converts stress to virial."""
+
+    # configs need to be in memory so they can be modified with stress -> virial, and safest to
+    # have them as a list (rather than using ConfigSet.to_memory()) when passing to ase.io.write below
+    fitting_configs = list(fitting_configs)
+
+    return fitting_configs
+    
+
+def fit(fitting_configs, mace_name, mace_fit_params, mace_fit_cmd, ref_property_prefix="REF_", 
+		skip_if_present=True, run_dir=".", verbose=True, do_fit=True, remote_info=None,
+        remote_label=None, wait_for_results=True, **kwargs):
+    """
+        Fit MACE model.
+    Parameters
+    ----------
+    fitting_configs: ConfigSet
+       set of configurations to fit 
+    mace_fit_params: str or dict
+        parameters for fitting the model, it can be directly read from YAML file or passed on as dict. 
+    mace_fit_cmd: str
+        command for excecuting the MACE fitting. (For example, "python ~/path_to_mace_cripts/run_train.py")
+    mace_name: str
+        name of MACE label
+    run_dir: str, default '.'
+        directory to run fitting in
+    remote_info: dict or wfl.autoparallelize.utils.RemoteInfo, or '_IGNORE' or None
+        If present and not None and not '_IGNORE', RemoteInfo or dict with kwargs for RemoteInfo
+        constructor which triggers running job in separately queued job on remote machine.  If None,
+        will try to use env var WFL_EXPYRE_INFO used (see below). '_IGNORE' is for
+        internal use, to ensure that remotely running job does not itself attempt to spawn another
+        remotely running job.
+    verbose: bool default True
+        verbose output
+    do_fit: bool, default True
+        carry out the fit, otherwise only print fitting command 
+    wait_for_results: bool, default True
+        wait for results of remotely executed job, otherwise return after starting job
+    remote_label: str, default None
+        label to match in WFL_EXPYRE_INFO
+    skip_if_present: bool, default False
```

Comment:
please make docstring arguments consistent (the same order, nothing missing, nothing extra) as the arguments in the function declaration. Specifically, I noticed that `kwargs` is undocumented (maybe other things, too - it's hard to tell when they're out of order.